build.gradle

```diff
@@ -186,6 +185,7 @@ def distContent = copySpec {
   into('build') {
     from "$buildDir"
     include '*.jar'
+    include 'i386/*.so'
```

Comment:
Currently there's no `i386` directory in the project. This line won't copy anything.
You want the `*.so` created by `make` copied to the build output?